person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if @age >= 18 || @parent_permission
```

Comment:
- [ ] @GutemaG kindly use your private method of_Age instead of repeating the same code, this will make your code clean, you also don't need to use true and false, as ruby will automatically execute the code depending on the condition given, eliminating need to write true or false you could write `of_age? || @parent_permission`.